src/models/validationModels/routePathLinkValidationModel.ts

```diff
@@ -0,0 +1,16 @@
+import Constants from '../../constants/constants';
+
+const routePathLinkValidationModel = {
+    id: `required|min:1|max:${Constants.INTEGER_MAX_VALUE}|numeric`,
+    routePathDirection: 'required|min:1|max:1|string',
+    routePathStartDate: 'required|date',
+    orderNumber: `required|min:1|max:${Constants.SMALL_INT_MAX_VALUE}|numeric`,
+    startNodeType: 'required|min:1|max:1|string',
+    transitType: 'required|min:1|max:1|string',
+    isStartNodeTimeAlignmentStop: 'boolean',
+    isStartNodeHastusStop: 'boolean',
+    isAtBookSchedule: 'boolean',
+    startNodeColumnNumber: 'numeric|max:99'
```

Comment:
min 0 ?

Reply:
Saving a large negative value is currently broken, and I don't see any reason why we should allow negative numbers